packages/jest-snapshot/src/index.js

```diff
@@ -100,21 +99,24 @@ const toMatchSnapshot = function(received: any, expected: void) {
       },
     );
 
-    const message =
-      () => matcherHint('.toMatchSnapshot', 'value', '') + '\n\n' +
-      `${RECEIVED_COLOR('Received value')} does not match ` +
+    const report =
+      () => `${RECEIVED_COLOR('Received value')} does not match ` +
```

Comment:
I'm not quite sure how to handle this particular bit. Report is very similar to the message - message only has one more line at the beginning that is implementation-specific.

I'm also confused that message is either a function or a string, depending on whether the test was successful or not.